From c69b818a9fce2bf0fb81b280d6f4cc15e6fdb8af Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 16 Dec 2022 17:13:43 +0000
Subject: [PATCH] GCC: fix incomplete type of gl::Presenter

GCC requires full definition of gl::Presenter for return type
of gl::GLOzoneEGL::CreateGLContext() with base::scoped_refptr.
---
 ui/ozone/common/gl_ozone_egl.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/ozone/common/gl_ozone_egl.cc b/ui/ozone/common/gl_ozone_egl.cc
index 9b1d37e..e745dd7 100644
--- a/ui/ozone/common/gl_ozone_egl.cc
+++ b/ui/ozone/common/gl_ozone_egl.cc
@@ -13,6 +13,7 @@
 #include "ui/gl/gl_share_group.h"
 #include "ui/gl/gl_surface.h"
 #include "ui/gl/gl_utils.h"
+#include "ui/gl/presenter.h"
 
 namespace ui {
 
-- 
2.37.4

